src/components/Admin/TrustedSources.js

```diff
@@ -17,24 +18,43 @@ class TrustedSources extends React.Component {
   }
 
   componentDidMount() {
-    this.props.flux.actions.ADMIN.load_trusted_sources(PIPELINE_KEYS);
+    this.props.flux.actions.ADMIN.notifyDataGridTrustedSourcesLoaded();
   }
 
   handleSave(rows) {
-    rows.map(row => this.appendRowKey(row));
+    rows.map(row => row.rowKey = this.getRowKey(row));
```

Comment:
Given that we're mutating values here and not transforming a list, should this be `forEach` instead of `map`?